CHANGES/909.bugfix.rst

```diff
@@ -1,3 +1,9 @@
 Revert to using the public argument parsing API
```

Comment:
Weird. Looks like I used a wrong style here too. It should be in the past tense.
```suggestion
Reverted to using the public argument parsing API
```